dev/io.openliberty.org.jboss.resteasy.cdi.ee10/src/io/openliberty/org/jboss/resteasy/common/component/LibertyResteasyCdiExtension.java

```diff
@@ -49,57 +50,32 @@ private boolean isSessionBean(AnnotatedType<?> annotatedType)
        }
        return false;
     }
-
+    
     /**
-     * Set a default scope for each CDI bean which is a RESTful WS Resource.
-     *
-     * @param <T> type
-     * @param event event
-     * @param beanManager bean manager
-     */
-    @Override
-    public <T> void observeResources(@WithAnnotations({Path.class}) @Observes ProcessAnnotatedType<T> event, BeanManager beanManager)
-    {
-       AnnotatedType<T> annotatedType = event.getAnnotatedType();
-       Class<?> javaClass = annotatedType.getJavaClass();
-       if(!javaClass.isInterface()
-           && !isSessionBean(annotatedType)
-           && !annotatedType.isAnnotationPresent(Decorator.class))
-       {
-          LogMessages.LOGGER.debug(Messages.MESSAGES.discoveredCDIBeanJaxRsResource(annotatedType.getJavaClass().getCanonicalName()));
-          //Liberty change start
-          if (Application.class.isAssignableFrom(javaClass)) {
-              event.setAnnotatedType(wrapAnnotatedType(annotatedType, applicationScopedLiteral));
-          } /* else {
-              event.setAnnotatedType(wrapAnnotatedType(annotatedType, requestScopedLiteral));
-          }
-          */
+    * Set a default scope for each CDI bean which is a RESTful WS Resource.
+    *
+    * @param <T>         type
+    * @param event       event
+    * @param beanManager bean manager
+    */
+   public <T> void observeResources(@WithAnnotations({Path.class}) @Observes ProcessAnnotatedType<T> event,
+                                    BeanManager beanManager) {
+      AnnotatedType<T> annotatedType = event.getAnnotatedType();
+      Class<?> javaClass = annotatedType.getJavaClass();
+      if (javaClass.isInterface()
```

Comment:
This is a major change that seems wrong.  Why is it changed to remove the `!`?

Reply:
Understanding this requires comparing Resteasy's Beta2 version of ResteasyCdiExtenstion to their Beta3/Final version.   This method mirrors what is done there.

Reply:
My mistake,  you were commenting on the "!" alone and not the amount of changes.   Thanks for spotting this error.

Reply:
Done